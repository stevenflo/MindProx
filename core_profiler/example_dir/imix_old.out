# FINI: end of program
Dynamic Instruction Count = 99232, Dyn BBL Count = 21998
<DynInsCount> 99232
<BBLCount>21998
<TotInsCount>99232
<IntCount> 34555
<IntMulCount > 26
<IntDivCount > 93
<FPCount> 0
<FPMulCount> 0
<FPDivCount> 0
<LdCount> 24629
<StCount> 12583
<Br/CallCount> 19427
<CondBrCount> 17901
<OtherCount> 64558
<SysCallCount> 40

# END_DYNAMIC_STATS
