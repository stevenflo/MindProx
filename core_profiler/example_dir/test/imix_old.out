# FINI: end of program
Dynamic Instruction Count = 96442, Dyn BBL Count = 21210
<DynInsCount> 96442
<BBLCount>21210
<TotInsCount>96442
<IntCount> 33378
<IntMulCount > 26
<IntDivCount > 93
<FPCount> 0
<FPMulCount> 0
<FPDivCount> 0
<LdCount> 24034
<StCount> 12427
<Br/CallCount> 18707
<CondBrCount> 17217
<OtherCount> 62945
<SysCallCount> 40

# END_DYNAMIC_STATS
